src/otx/algo/model/__init__.py

```diff
@@ -0,0 +1 @@
+"""Module for OTX custom model."""
```

Comment:
What is the purpose of this folder? It's currently empty.

Reply:
See `for_developers/dir_structure.md`. It is reserved for the OTX custom algo (e.g. models built upon `nn.Module`).